test/System/IntegrationTests/IntegrityTests.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using NUnit.Framework;
+
+namespace IntegrationTests
+{
+    [TestFixture]
+    class IntegrityTests
+    {
+        [SetUp]
+        public void Setup()
+        {
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+        }
+
+        /// <summary>
+        /// This test scans "Dynamo/src" and sub-folders for "Resources.resx"
+        /// files, and tries to compare them with their "Resources.en-US.resx" 
+        /// counterparts to make sure they are exactly the same. The guideline 
+        /// is, everytime "Resources.resx" is updated, it should be copied to 
+        /// overwrite "Resources.en-US.resx" so that they are always in-sync.
+        /// </summary>
+        /// 
+        [Test]
+        public void AllResxFilesShouldMatchEnUsCounterparts()
+        {
+            var executingAsmPath = Assembly.GetExecutingAssembly().Location;
```

Comment:
This test may not work on CI system, sometimes we just copy binaries to run the tests. On the test machine, we may not have the source code to run this test.